src/sentry/tagstore/snuba/backend.py

```diff
@@ -422,17 +424,24 @@ def get_group_seen_values_for_environments(
             ["max", SEEN_COLUMN, "last_seen"],
         ]
 
-        result = snuba.query(
+        result = snuba.aliased_query(
+            dataset=snuba.Dataset.Events,
             start=start,
             end=end,
             groupby=["group_id"],
-            conditions=conditions,
+            conditions=snuba_filters,
             filter_keys=filters,
             aggregations=aggregations,
             referrer="tagstore.get_group_seen_values_for_environments",
         )
 
-        return {issue: fix_tag_value_data(data) for issue, data in six.iteritems(result)}
+        return OrderedDict(
```

Comment:
Why does the key ordering matter here?

Reply:
Yes, looks like it is just a leftover statement

Reply:
Iterating over results constructs an `OrderedDict` in the previous implementation here, I wanted to keep the return type same

Reply:
Reverted back to `dict` implementation